lib/client/db/dbcmd/error.go

```diff
@@ -46,6 +46,12 @@ func ConvertCommandError(cmd *exec.Cmd, err error, peakStderr string) error {
 		}
 	}
 
+	lowerCaseStderr := strings.ToLower(peakStderr)
+	if strings.Contains(lowerCaseStderr, "access to db denied") {
```

Comment:
Can you add a UT that covers this scenario?

Reply:
So just a test of ConvertCommandError input/output?

Reply:
Ok simple enough, added